src/PointwiseFunctions/AnalyticSolutions/Elasticity/BentBeam.hpp

```diff
@@ -0,0 +1,150 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <limits>
+
+#include "DataStructures/DataBox/Prefixes.hpp"   // IWYU pragma: keep
+#include "DataStructures/Tensor/Tensor.hpp"      // IWYU pragma: keep
+#include "Elliptic/Systems/Elasticity/Tags.hpp"  // IWYU pragma: keep
+#include "Options/Options.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/IsotropicHomogeneous.hpp"
+#include "PointwiseFunctions/Elasticity/ConstitutiveRelations/Tags.hpp"  // IWYU pragma: keep
+#include "Utilities/TMPL.hpp"
+#include "Utilities/TaggedTuple.hpp"
+
+// IWYU pragma: no_forward_declare Tensor
+
+/// \cond
+class DataVector;
+namespace PUP {
+class er;
+}  // namespace PUP
+/// \endcond
+
+namespace Elasticity {
+namespace Solutions {
+/*!
+ * \brief A state of pure bending of an elastic beam in 2D
+ *
+ * \details This solution describes a 2D slice through an elastic beam of length
+ * \f$L\f$ and height \f$H\f$ that is subject to a bending moment \f$M=\int
+ * T^{xx}y\mathrm{d}y\f$ (see e.g. \cite ThorneBlandford2017, Eq. 11.41c for a
+ * bending moment in 1D). The beam material is characterized by an isotropic and
+ * homogeneous constitutive relation \f$Y^{ijkl}\f$ in the plane-stress
+ * approximation (see
+ * `Elasticity::ConstitutiveRelations::IsotropicHomogeneous`). In this scenario,
+ * no body-forces \f$f_\mathrm{ext}^j\f$ act on the material, so the
+ * \ref Elasticity equations reduce to \f$\nabla_i T^{ij}=0\f$, but the bending
+ * moment \f$M\f$ generates the stress
+ * \f[
```

Comment:
To avoid clang-format causing problems in the future for the equations could you please put a blank line before the `\f[`, `\f{align}` and after `\f[`, `\f}`?